packages/select/src/vaadin-select.js

```diff
@@ -170,6 +172,26 @@ class Select extends DelegateFocusMixin(FieldMixin(SlotMixin(ElementMixin(Themab
 
   static get properties() {
     return {
+      /**
+       * An array containing items that will be rendered as the select's options.
+       *
+       * #### Example
+       * ```js
+       * select.items = [
+       *   { label: 'Jose', value: 'jose' },
+       *   { component: 'hr' },
+       *   { label: 'Pedro', value: 'pedro' },
+       *   { label: 'Manolo', value: 'manolo', disabled: true }
+       * ];
+       * ```
+       * @type {!Array<!SelectItem>}
+       */
+      items: {
+        type: Array,
+        value: () => [],
```

Comment:
`items` property should be `undefined` initially. Same as with Grid or Combo Box, in case you use a `dataProvider` instead of `items` https://vaadin.com/components/vaadin-combo-box/html-examples/combo-box-basic-demos#lazy-loading

Reply:
It seems that I added the default value because I followed the menu-bar as an example. Good point, I'll change it now.

Reply:
Fixed. Please, check it out.